End of year 2022 post (#167)
ARCLeeds/arcleeds.github.io: blog/_posts/2022-12-16-endofyear2022.md

---
layout: page
title: "A year in review: 2022"
date: 2022-12-15
subheadline: "That's a wrap on 2022!"
teaser: "
 A final post from the team looking back on 2022.
"
author: "Research Computing"

image:
  thumb: HPC3small.jpg

breadcrumb: true
---

As the year draws to a close the team reflect on what they've been up to this year.

## Research IT Strategy and Business Case

The future is bright for Research IT at Leeds! The University has made a major commitment to research by signing off the Research IT Strategy and Business Case with a substantial investment to deliver new research IT services over the next planning period.

We will have more on that in 2023, if you'd like to helped get involved in the next steps of this work [get in touch](https://it.leeds.ac.uk/it?id=sc_cat_item&sys_id=7587b2530f675f00a82247ece1050eda)!

## ResCompLeedsCon

> This year we organised, with the help of a fabulous organising committee, the first ever [Research Computing Leeds Conference](https://rescompleedscon.github.io/conference2022/). This brought together researchers from across the University who did computational research in some form and allowed them to tell their story and share best practice. It was held over 2 days with 2 streams of activity with an exciting range of talks, workshop session and panel discussions on Open Research and Sustanability and Research Computing.

Reviewer: ```suggestion
This year we organised, with the help of a fabulous organising committee, the first ever [Research Computing Leeds Conference](https://rescompleedscon.github.io/conference2022/). This brought together researchers from across the University who did computational research in some form and allowed them to tell their story and share best practices. It was held over 2 days with 2 streams of activity with an exciting range of talks, workshop session and panel discussions on Open Research and Sustainability and Research Computing.
```

The conference was well received and we hope to be able to run it again for 2023! If you'd like to get involved [get in touch](https://it.leeds.ac.uk/it?id=sc_cat_item&sys_id=7587b2530f675f00a82247ece1050eda).

<div style="text-align:center">

  <img alt="David Hogg giving keynote talk at ResCompLeedsCon2022" src="/images/blog/endofyear2022/rescompleedscon-davidhogg.jpg">
      <figcaption>
        Professor David Hogg giving keynote talk at ResCompLeedsCon2022.
      </figcaption> 
</div>

## Researcher Training

Over the course of the year we deliver regular training sessions from our [portfolio of courses](https://arc.leeds.ac.uk/training/courses/). This year many of these sessions returned to an in-person format and we were excited to develop and run some new courses such as a revamped [SWD6: High Performance Python](https://arctraining.github.io/swd6_hpp/) and [HPC2: Managing and installing applications on the HPC](https://arctraining.github.io/hpc2-software/welcome.html). These courses were developed and hosted as GitHub pages and you can review all the course material via those links.

> Overall, we delivered approximately 30 courses to over 500 researchers on computational techniques and best practices. We'll be back with more sessions in 2023 and also have other new courses in development!

Reviewer 2: ```suggestion
Overall, we delivered approximately 30 courses to over 500 researchers on computational techniques and best practices. We'll be back with more sessions in 2023 and have new courses and updates in development!
```

## HPC

We can't reflect on the year without talking a little bit about HPC! As of writing researchers have racked up an impressive 3,982,631 computational jobs across both ARC3 and ARC4. 

You can read about some highlights of the work performed using the HPC via our [12 days of HPC blog series for 2023](https://arc.leeds.ac.uk/blog/).

## Research Software Engineering projects

Alongside training and HPC we've also supported a number of research projects this year from a variety of departments. These have ranged from actively debugging Shiny Apps to providing research software engineering best practice support to academics starting computational research projects.

> If you'd like to explore getting RSE support for work you are doing or have planned why not [get in touch](https://it.leeds.ac.uk/it?id=sc_cat_item&sys_id=7587b2530f675f00a82247ece1050eda)?

Reviewer 2: ```suggestion
RSE project work is already being scheduled for 2023, if you'd like to explore getting RSE support for work you are doing or have planned why not [get in touch](https://it.leeds.ac.uk/it?id=sc_cat_item&sys_id=7587b2530f675f00a82247ece1050eda)?
```
